assets/js/components/common/Debug.jsx

```diff
@@ -114,28 +165,31 @@ class Debug extends Component {
             icon={<ReloadOutlined />}
             shape="circle"
             onClick={() => {
-              this.setState({ data: [] })
+              this.setState({ data: [] });
             }}
           />
         </div>
         <div style={{ width: "100%", marginTop: 50 }}>
-          {
-            data.map(d => (
-              <span key={d.id}>
-                <DebugEntry event={d} clearSingleEntry={this.clearSingleEntry} expandAll={expandAll} />
-              </span>
-            ))
-          }
+          {data.map((d) => (
+            <span key={d.id}>
+              <DebugEntry
+                event={d}
+                clearSingleEntry={this.clearSingleEntry}
+                expandAll={expandAll}
+                width={this.props.entryWidth || 600}
```

Comment:
main change in this file to allow for a smaller debug entry when opening from node workspace

Reply:
ty